code/game/gamemodes/events/biomass.dm

```diff
@@ -202,10 +202,11 @@
 		var/area/Hallway = locate(type)
 
 		for(var/turf/simulated/floor/Floor in Hallway.contents)
-			if(Floor.contents.len <= 0)
+			if(Floor.contents.len <= 1)//There's always an /atom/movable/lighting_overlay on turfs

```

Comment:
Yes I do agree it *did* use that but it also used the extremely poor contents check so I don't believe it was thought out very well. This method would help prevent vines from spawning within a window and doing nothing as well as an added bonus to not breaking entirely the second someone changes something.